aws_kms_pkcs11.cpp

```diff
@@ -180,8 +203,7 @@ CK_RV C_Initialize(CK_VOID_PTR pInitArgs) {
             Aws::KMS::Model::ListKeysOutcome res = kms.ListKeys(req);
             if (!res.IsSuccess()) {
                 debug("Got error from AWS list keys: %s", res.GetError().GetMessage().c_str());
-                C_Finalize(NULL_PTR);
-                return CKR_FUNCTION_FAILED;
+                break;
```

Comment:
I feel like we should set `result = CKR_FUNCTION_FAILED;` here.

Reply:
Ack